Model/System/Config.php

```diff
@@ -69,7 +69,7 @@ public function isCountryInEu($countryCode)
      */
     public function getEuCountries()
     {
-        $euCountries = $this->scopeConfig->getValue('general/country/eu_countries');
+        $euCountries = $this->scopeConfig->getValue('general/country/eu_countries', 'store');
```

Comment:
Please use `\Magento\Store\Model\ScopeInterface::SCOPE_STORE` instead.

Reply:
I have changed it and also file \FireGento\MageSetup\Block\Imprint\Content